### DISCUSSION TITLE
Problem with connecting my java profect with json

### DISCUSSION THREAD

**Discussion by LukasGer-byte** [community/community#71981] [Programming Help]
### Body

Hi i have a problem with my java code. I wanted to connect a json file to my java project but i get a error while running.

Here is the Code:
``` 
try {

        String k = "{\"username\":\"patric\",\"userpassword\":abcd}";
        Object file = JSONValue.parse(k);


        JSONObject jsonObjectdecode = (JSONObject)file;


        String name
            = (String)jsonObjectdecode.get("Full Name");
        String password
            = (String)jsonObjectdecode.get("Tuition Fees");
        System.out.println(name + " " + password);

        } catch (Exception ex) {
            ex.printStackTrace();
        }
```        
The Error:
```
java.lang.NullPointerException: Cannot invoke "org.json.simple.JSONObject.get(Object)" because "jsonObjectdecode" is null
	at Main$VSystem.menu(Main.java:151)
	at Main.main(Main.java:81)
```

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebubekirgungor** [comment]
Hello, you should put "abcd" in quotes like: `String k = "{\"username\":\"patric\",\"userpassword\":\"abcd\"}";`

**Comment by Brazlucas** [comment]
Try this:

 try {
    String k = "{\"username\":\"patric\",\"userpassword\":\"abcd\"}";
    Object file = JSONValue.parse(k);

    JSONObject jsonObjectdecode = (JSONObject) file;

    String username = (String) jsonObjectdecode.get("username");
    String userpassword = (String) jsonObjectdecode.get("userpassword");
    
    System.out.println("Username: " + username);
    System.out.println("Password: " + userpassword);

} catch (Exception ex) {
    ex.printStackTrace();
}

**Comment by selin-topcu** [comment]
Can you try?
```
try {
    String k = "{\"username\":\"patric\",\"userpassword\":\"abcd\"}";
    Object file = JSONValue.parse(k);
    JSONObject jsonObjectdecode = (JSONObject)file;

    String name = (String)jsonObjectdecode.get("username");
    String password = (String)jsonObjectdecode.get("userpassword");
    System.out.println(name + " " + password);
} catch (Exception ex) {
    ex.printStackTrace();
}
```
